grammars/css.cson

```diff
@@ -528,219 +1612,227 @@
-        'name': 'entity.name.tag.css'
+        'include': '#selector-innards'
       }
+    ]
+  'selector-innards':
+    'patterns': [
       {
-        'match': '\\*'
-        'name': 'entity.name.tag.wildcard.css'
+        'match': '&'
+        'name': 'keyword.operator.nesting.css'
       }
       {
-        'captures':
-          '1':
-            'name': 'punctuation.definition.entity.css'
-        'match': '''(?x)
-          (\\.)
-          -?+                       # possessive to avoid matching '-' followed by [0-9]
-          [-a-zA-Z_\\P{ASCII}]
-          [-a-zA-Z0-9_\\P{ASCII}]*+
-          (?=$|[\\s,.:{>+~|])       # to avoid matching '.B' in '.B&W'
-        '''
-        'name': 'entity.other.attribute-name.class.css'
+        'include': '#comment-block'
       }
       {
         'captures':
           '1':
-            'name': 'punctuation.definition.entity.css'
-        'match': '''(?x)
-          (\\#)
-          -?+[-a-zA-Z_\\P{ASCII}][-a-zA-Z0-9_\\P{ASCII}]*+
-          (?=$|[\\s,.:{>+~|])
-        '''
-        'name': 'entity.other.attribute-name.id.css'
+            'name': 'keyword.control.at-rule.import.css'
+          '2':
+            'name': 'punctuation.definition.keyword.css'
+        'match': '(?i)((@)nest)(?![\\w-])\\s*'
+      }
+      {
+        'include': '#escapes'
+      }
+      {
+        'include': '#combinators'
       }
       {
         'captures':
           '1':
-            'name': 'punctuation.definition.entity.css'
-          '2':
             'name': 'entity.other.namespace-prefix.css'
-          '3':
+          '2':
             'name': 'punctuation.separator.css'
-          '4':
-            'name': 'entity.other.attribute-name.css'
-          '5':
-            'name': 'keyword.operator.pattern.css'
-          '6':
-            'name': 'string.unquoted.attribute-value.css'
-          '7':
-            'name': 'string.quoted.attribute-value.css'
-          '8':
-            'name': 'punctuation.definition.string.begin.css'
-          '9':
-            'name': 'punctuation.definition.string.end.css'
-          '10':
-            'name': 'punctuation.definition.entity.css'
         'match': '''(?x)
-          (\\[)\\s*
-          (?:(\\g<ident>|\\*)?(\\|))?     # qualified name prefix
-          ([\\w:&&[^0-9]][\\w:.-]*+)\\s*  # attribute name
-          (?:
-            ([~|^$*]?=)\\s*
-            (?:                           # attribute value
-              (\\g<ident>)
-            | ((?>([\'"])(?:[^\\\\]|\\\\.)*?(\\k<-2>)))
-            )\\s*
+          (?:^|(?<=[\\s,(};]))         # Follows whitespace, comma, semicolon, or bracket
+          (?!
```

Comment:
... for checking if there's a selector after the `|`. If you delete the group and run the specs, you'll see.